backend/src/datasources/postApi.js

```diff
@@ -0,0 +1,44 @@
+const { RESTDataSource } = require('apollo-datasource-rest');
+
+class PostsAPI extends RESTDataSource {
+
+  constructor() {
+    super();
+    this.posts = [
+      {
+        title: 'The Thing',
+        votes: 0,
+        author: 'Peter',
+        upvoters:[]
+      },
+      {
+        title: 'The Nothing',
+        votes: 0,
+        author: 'Peter',
+        upvoters:[]
+      },
+    ];
```

Comment:
```suggestion
    this.posts = []
```
Seed the database when needed, ie. when you start the server for development. Use something like `seed(myDataSource)`.